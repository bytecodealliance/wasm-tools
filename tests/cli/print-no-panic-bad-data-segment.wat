;; RUN: print %

(module
    (data (br 4294967295) "")
)
