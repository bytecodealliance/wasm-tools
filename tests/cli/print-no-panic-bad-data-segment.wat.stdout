(module
  (data (;0;) (br 4294967295 (; INVALID ;)) "")
)
